src/functions/template-tags/general.php

```diff
@@ -1202,7 +1202,24 @@ function tribe_event_featured_image( $post_id = null, $size = 'full', $link = tr
 }
 
 /**
- * Return the details of the start/end date/time.
+ * Capitalizes meridiem strings (am/pm) in time strings.
+ *
+ * Takes a formatted time string and ensures that any meridiem indicators
+ * (am/pm or a.m./p.m.) are capitalized to AM/PM or A.M./P.M. respectively.
+ * This maintains consistency in time display across the plugin.
+ *
+ * @since TBD
+ *
+ * @param string $formatted_time The time string potentially containing am/pm indicators.
+ *
+ * @return string The time string with capitalized AM/PM if present.
+ */
+function tec_events_capitalize_meridiem( $formatted_time ) {
```

Comment:
This approach is too loose. It is not location-aware.
ex: in Albanian, they can use "jam" as "am" (although most of Europe typically uses 24-hour time) and this would turn that into "jAM"

Reply:
I would look at 
1) addressing the `@todo` and using tribe_get_datetime_format and/or tribe_get_time_format
2) adapting those to capitalize the "a" in "g:i a", etc